x-pack/plugins/secops/public/components/page/network/network_dns_table/index.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { getOr } from 'lodash/fp';
+import * as React from 'react';
+import { MockedProvider } from 'react-apollo/test-utils';
+import { Provider as ReduxStoreProvider } from 'react-redux';
+
+import { mockGlobalState, TestProviders } from '../../../../mock';
+import { createStore, networkModel, State } from '../../../../store';
+
+import { NetworkDnsTable } from '.';
+import { mockData } from './mock';
+
+describe('NetworkTopNFlow Table Component', () => {
+  const loadMore = jest.fn();
+  const state: State = mockGlobalState;
+
+  let store = createStore(state);
+
+  beforeEach(() => {
+    store = createStore(state);
+  });
+
+  describe('rendering', () => {
+    test('it renders the default NetworkTopNFlow table', () => {
+      const wrapper = shallow(
+        <ReduxStoreProvider store={store}>
+          <NetworkDnsTable
+            loading={false}
+            data={mockData.NetworkDns.edges}
+            totalCount={mockData.NetworkDns.totalCount}
+            hasNextPage={getOr(false, 'hasNextPage', mockData.NetworkDns.pageInfo)!}
+            nextCursor={getOr(null, 'endCursor.value', mockData.NetworkDns.pageInfo)!}
+            loadMore={loadMore}
+            startDate={1546965070707}
```

Comment:
For tests around epoch's I would use 

```ts
new Date('2019-01-08T16:31:10.707Z').valueOf()
```
so its human readable of what date time is being used in the test for future humans/ai robot overlords who might be parsing this. Well, if it's robot overlords, we'll probably be fine.